db/comdb2.h

```diff
@@ -1182,27 +1176,19 @@ typedef struct bpfunc_lstnode {
 typedef LISTC_T(bpfunc_lstnode_t) bpfunc_list_t;
 /*******************************************************************/
 
-enum OSQL_REQ_TYPE {
-    OSQL_REQINV = 0,
-    OSQL_BLOCK_REQ = 1, /* obsolete */
-    OSQL_SOCK_REQ = 2,
-    OSQL_RECOM_REQ = 3,
-    OSQL_SERIAL_REQ = 4,
-
-    OSQL_BLOCK_REQ_COST = 5, /* obsolete */
-    OSQL_SOCK_REQ_COST = 6,
-
-    OSQL_SNAPISOL_REQ = 7,
-    OSQL_SNAP_UID_REQ = 8,
-    OSQL_MAX_REQ = 9,
-};
+typedef enum {
+    OSQL_REQINV,
+    OSQL_SOCK_REQ,
+    OSQL_RECOM_REQ,
+    OSQL_SERIAL_REQ,
+    OSQL_SNAPISOL_REQ,
+    OSQL_MAX_REQ
+} OSQL_REQ_TYPE;
```

Comment:
you cannot remap values, can you? recom will be treated as sock, and so on

Reply:
These are never sent on the wire. There is no need to keep obsolete values around.

Reply:
OK, turns out your `sockbplog` does use it. But since we don't run that code, now is a good time to remedy it. I have made it use the `NET` type and derive `OSQL` type from it like rest of system does.